tensorrt_llm/_torch/visual_gen/models/hunyuan_video1_5/transformer_hunyuan_video1_5.py

```
@@ -0,0 +1,1209 @@
+            encoder_query = encoder_query.unflatten(2, (self.heads, -1))
+            encoder_key = encoder_key.unflatten(2, (self.heads, -1))
+            encoder_value = encoder_value.unflatten(2, (self.heads, -1))
+
+            encoder_query = self._head_norm(self.norm_added_q, encoder_query)
+            encoder_key = self._head_norm(self.norm_added_k, encoder_key)
+
+            query = torch.cat([query, encoder_query], dim=1)
+            key = torch.cat([key, encoder_key], dim=1)
+            value = torch.cat([value, encoder_value], dim=1)
+
+        batch_size, seq_len, *_ = query.shape
```

Ruff (0.16.1)

[warning] Unpacked variable `batch_size` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)